sw/device/silicon_creator/manuf/base/provisioning_inputs.bzl

```diff
@@ -2,11 +2,42 @@
 # Licensed under the Apache License, Version 2.0, see LICENSE for details.
 # SPDX-License-Identifier: Apache-2.0
 
-EARLGREY_INDIVIDUALIZE_OTP_SW_CFGS = [
+EARLGREY_OTP_CFGS = [
     "sival",
     "prodc",
 ]
 
+EARLGREY_SKUS = {
+    # OTP Config: SIVAL; DICE Certs: X.509; Additional Certs: None
+    "sival": {
+        "otp": "sival",
+        "dice_libs": ["//sw/device/silicon_creator/lib/cert:dice"],
+        "ext_libs": ["@provisioning_exts//:perso_fw_ext"],
+    },
+    # OTP Config: SIVAL; DICE Certs: CWT; Additional Certs: None
+    # TODO(#24281): uncomment when DICE CWT cert flows are fully supported
```

Comment:
FYI: @tommychiu-github @stevenchtsai 

This flow does not work yet because the host test harness does not yet know how to parse CWTs and endorse them. You can uncomment this once you complete the CWT work.

Reply:
Got it, thanks for the heads up.